examples/lighting-app/qpg/src/LightingManager.cpp

```diff
@@ -21,12 +21,31 @@
 #include "qvCHIP.h"
 #include <support/logging/CHIPLogging.h>
 
+// initialization values for Blue in XY color space
+#define BLUE_XY                                                                                                                    \
```

Comment:
please define a `constexpr XyColor_t kBlueXy = { 9830, 3932 }` (or constexpr kBlueXyX and similar) instead and similar below. This has the benefit of being typesafe and prefering less macros on cpp code (easier to read)